rust/parquet/README.md

```diff
@@ -46,6 +46,15 @@ while let Some(record) = iter.next() {
 ```
 See [crate documentation](https://docs.rs/crate/parquet/4.0.0-SNAPSHOT) on available API.
 
+## Upgrading from versions prior to 4.0
+
+*Upgrade Note*: If you used version 3.0 or previous of this crate, you
+likely need to change your code to refer to [`ConvertedType`] rather
+than [`LogicalType`]. Version 4.0 introduces an *entirely new* struct
+called `LogicalType` to align with the standard, and `LogicalType` was
```

Comment:
nit: to align with the standard -> to align with the `LogicalType` introduced in Parquet Format 2.4.0 and up.